adapters/openx/openx.go

```diff
@@ -298,3 +306,51 @@ func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server co
 	}
 	return bidder, nil
 }
+
+func getBidMeta(bid *openrtb2.Bid) *openrtb_ext.ExtBidPrebidMeta {
+	if bid.Ext == nil {
+		return nil
+	}
+
+	var ext *oxBidExt
+	if err := jsonutil.Unmarshal(bid.Ext, &ext); err != nil {
+		return nil
+	}
+
+	buyerId := getBuyerIdFromExt(ext)
+	dspId := getDspIdFromExt(ext)
+	brandId := getBrandIdFromExt(ext)
+	if buyerId <= 0 && dspId <= 0 && brandId <= 0 {
+		return nil
+	}
+
+	return &openrtb_ext.ExtBidPrebidMeta{
+		NetworkID:    dspId,
+		AdvertiserID: buyerId,
+		BrandID:      brandId,
+	}
+}
+
+func getBuyerIdFromExt(ext *oxBidExt) int {
```

Comment:
this function will work correctly
but still it's best practice to check for an empty value before passing value to strconv.Atoi() to avoid spawning an error each time there is no value available in json

```suggestion
func getBuyerIdFromExt(ext *oxBidExt) int {
	if ext.BuyerId == "" {
		return 0
	}	
```

Reply:
added, thanks for the suggestion